server_setup.coffee

```diff
@@ -90,15 +90,7 @@ setupExpressMiddleware = (app) ->
 
   app.use require('serve-favicon') path.join(__dirname, 'public', 'images', 'favicon', 'favicon.ico')
   app.use require('cookie-parser')()
-  app.use require('body-parser').json({limit: '25mb', strict: false, verify: (req, res, buf, encoding) ->
-    if req.headers['x-hub-signature']
-      # this is an intercom webhook request, with signature that needs checking
-      try
-        digest = crypto.createHmac('sha1', config.intercom.webhookHubSecret).update(buf).digest('hex')
-        req.signatureMatches = req.headers['x-hub-signature'] is "sha1=#{digest}"
-      catch e
-        log.info 'Error checking hub signature on Intercom webhook: ' + e
-  })
+  app.use require('body-parser').json limit: '25mb', strict: false
   app.use require('body-parser').urlencoded extended: true, limit: '25mb'
   app.use require('method-override')()
   app.use require('cookie-session')
```

Comment:
This looks great! I wonder if most of this can be deleted from the frontend repo. We only need to preserve the proxy part for local testing.

Not required for this PR though!